test/evergreen/find_gperftools.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
```

Comment:
It would be useful to add a description of what this script does at the top of the file. The `find_gperftools` function name is misleading as it also will download the archive and install it. Please add a comment to explain the input parameters, their valid values, and an example of usage. Comments might be useful as well.

Reply:
Added in comments and restructured the script a little bit. Let me know if there is more improvement that we need.